homeassistant/components/poolstation/translations/es.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Only the generated `en.json` translation file is allowed. Don't add other translations in a PR.
Translations are handled by Lokalise, this will override the `es.json` file.
https://developers.home-assistant.io/docs/translations

Reply:
Gotcha.